config/canonical.json

```diff
@@ -22249,8 +22249,8 @@
     ],
     "tags": {
       "brand": "Federated Co-operatives",
-      "brand:wikidata": "Q5440676",
-      "brand:wikipedia": "en:Federated Co-operatives",
+      "brand:wikidata": "Q3277439",
+      "brand:wikipedia": "en:Co-op Food",
```

Comment:
Based on https://github.com/osmlab/name-suggestion-index/issues/2315 both en:Federated Co-operatives and en:Co-op Food are correct, just in different areas.

So both should be present.

Reply:
Doing this is a bit tricky - see https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md#identical-names-multiple-brands

Reply:
Interesting. Thank you for the reference. Ill check out it out again and make the needed changes best I can.

Reply:
Unfortunately this issue is a bit trickier than a typical one, what is a bit unlucky for a first issue :(

I wish you a good luck and feel free to ask if documentation is unclear - or even better, open a new issue (unclear documentation is a problem that should be fixed).